lib/AST/ClangTypeConverter.cpp

```diff
@@ -865,6 +865,9 @@ ClangTypeConverter::getClangTemplateArguments(
     ArrayRef<Type> genericArgs,
     SmallVectorImpl<clang::TemplateArgument> &templateArgs) {
   assert(templateArgs.size() == 0);
+  if (genericArgs.empty()) // If we have no generic argument, this is a noop.
```

Comment:
If we have a comment here can we explain why it is a noop? I feel like `genericArgs.empty()` communicates that sort of on it own?

Reply:
There is actually a comment that should go here. You're right, it's not the comment that is currently there, though. I added a different comment explaining why we *have* to bail here so the assertion directly under this doesn't fire. Let me know what you think.